fabric_plugin/tests/blueprint/blueprint.yaml

```diff
@@ -13,6 +13,7 @@ inputs:
     custom_input:
         default: value
     use_sudo:        {}
+    non_recoverable_errors:     {}
```

Comment:
The name is bit confusing, maybe use :
non_recoverable_error_exit_codes
 what do you think ?
@EarthmanT

Reply:
Sounds good. Shouldn't it be a list?

Reply:
@ahmadiesa-abu